packages/react/package.json

```diff
@@ -271,14 +325,14 @@
     "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-uglify": "^6.0.4",
     "sass": "^1.34.0",
-    "sass-loader": "^7.1.0",
     "storybook-addon-rtl": "^0.2.2",
     "style-loader": "^0.23.1",
     "stylelint": "^10.1.0",
     "stylelint-config-recommended-scss": "^3.3.0",
     "stylelint-scss": "^3.10.0",
     "weak-napi": "^2.0.1",
-    "webpack": "^4.28.4",
+    "webpack": "^4.41.5",
```

Comment:
I get some additional changes in the yarn.lock file when I pulled it down locally and ran `yarn install` might want to confirm everything is in there.